### PR TITLE
Networking/vhost rebase

### PR DESCRIPTION
**Pull request by egernst**
-update to 2.2.1 baseline
-edit version description which is printed in order to easily identify this is a networking-poc runtime

### CODE REVIEW

File: src/commands/version.c [hunk 1]
```diff
@@ -23,7 +23,7 @@
 static void
 show_version (void)
 {
-	g_print ("%s version: %s\n", PACKAGE_NAME, PACKAGE_VERSION);
+	g_print ("%s version: %s-networking-poc\n", PACKAGE_NAME, PACKAGE_VERSION);
```
Comment (mcastelino) [inline src/commands/version.c]: This is the only change right.

LGTM

**Comment by egernst** [comment]
This change and the rebase to latest master branch.

File: src/commands/version.c (hunk 1 above)
Comment (jodh-intel) [inline src/commands/version.c]: If that is indeed the only non-rebase change (please could you specify in subsequent PRs to make life easier)...

lgtm.

**Comment by jodh-intel** [comment]
@egernst - I'm afraid you've been bitten by the Travis bug(s). You have two options afaik:

- Make a trivial change to force a git commit hash change force push to update this PR.
- If that doesn't work, close and re-open the PR :-( (see https://github.com/travis-ci/travis-ci/issues/7830).

**Comment by egernst** [comment]
@jodh-intel : Sure, will do the fixup.  re:  (please could you specify in subsequent PRs to make life easier) --> I mentioned the changes in the pull request already, right?  Where would be easier to add this description?

**Comment by jodh-intel** [comment]
Hi @egernst - yes, I saw that you'd put a brief not on the PR, but it would be helpful to make it slightly clearer that the only change you'd actually made to the branch was the 1 line `src/commands/version.c` change. Clearly, @mcastelino highlighted that but it might have taken a bit of trawling through those 89 commits if he hadn't got here first ;)

**Pull request merged**